source/jormungandr/jormungandr/fallback_modes.py

```diff
@@ -89,3 +89,12 @@ def get_allowed_combinations_str(cls):
 all_fallback_modes = FallbackModes.modes_str()
 
 allowed_combinations = FallbackModes.get_allowed_combinations_str()
+
+street_network_modes_in_db = {
+    FallbackModes.car.name: "street_network_car",
+    FallbackModes.walking.name: "street_network_walking",
+    FallbackModes.bike.name: "street_network_bike",
+    FallbackModes.bss.name: "street_network_bss",
+    FallbackModes.ridesharing.name: "street_network_ridesharing",
+    FallbackModes.taxi.name: "street_network_taxi",
+}
```

Comment:
These names are supposed to be the id in the db? Isn't there an enum for them to use here?
Or is it the column name?

Reply:
My need here is to have a safe way to match the mode use for the request with the db column in instance. For example "car" = "street_network_car"